test/test_alarm.cpp

```diff
@@ -294,60 +294,153 @@ TEST_F (AlarmUnitTest, AlarmCheckAddDiagItemToSummary)
       PNET_DIAG_CH_PROP_MAINT_FAULT);
 }
 
-
-TEST_F (AlarmUnitTest, AlarmCheckQueueHandling)
+TEST_F (AlarmUnitTest, AlarmCheckSendQueueHandling)
 {
-   pf_alarm_queue_t queue;
+   pf_alarm_send_queue_t queue;
    pf_alarm_data_t alarm_input;
    pf_alarm_data_t alarm_output;
    int ix = 0;
    int err = 0;
    const uint16_t SEQUENCE_START_NUMBER = 100;
 
    /* Prepare default data */
-   memset (&queue, 0, sizeof (queue));
    memset (&alarm_input, 0, sizeof (alarm_input));
    memset (&alarm_output, 0, sizeof (alarm_output));
-   EXPECT_EQ (queue.count, 0);
+
```

Comment:
Check if there is boilerplate setup code here that could be moved to the test fixture.

Reply:
There are different types for the arguments in the different test cases, so there is not much gain to move to the fixture.
Clarified this by giving better names.
Done.